scripts/build_remote.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
```

Comment:
Is this used anywhere? If not maybe we should make this a follow up

Reply:
I've been using this to build on windows and it's been helpful for iterative dev/test cycles.